site/.firebaserc

```diff
@@ -0,0 +1,5 @@
+{
+  "projects": {
+    "default": "hnpwa-site"
```

Comment:
Does Firebase hosting support adding permissions for multiple folks to access config for this FB project through the console? Asking as it would be good to know if we'll need to chase you for help each time :)